### PR TITLE
fill color customization

### PR DESCRIPTION
**Pull request by bipinkrish**

### CODE REVIEW

File: signwriting/visualizer/test_visualize.py [hunk 1]
```diff
@@ -55,10 +54,9 @@ def test_image_small_box_is_corrected(self):
         assert_array_equal(np.array(image), np.array(reference_image))
 
     def test_image_with_line_and_embedded_color(self):
-        fsw = "M518x518S30a00482x483S33e00482x483"
-
-        image = signwriting_to_image("M518x518S30a00482x483S33e00482x483",
-                                     line_color=(144, 70, 180, 255), embedded_color=True)
+        fsw = "M530x518S19a30500x482S19a38465x481S22f04509x506S22f14467x504"
+        image = signwriting_to_image(fsw, line_color=(
+            10, 23, 122, 255), fill_color=(255, 255, 255, 0))
```
Comment (AmitMY) [inline signwriting/visualizer/test_visualize.py]: i don't like where the line breaks - either new line before `line_color` or same line with the color

File: signwriting/visualizer/visualize.py [hunk 2]
```diff
@@ -20,9 +21,10 @@ def get_symbol_size(symbol: str):
     return right - left, bottom - top
 
 
-# pylint: disable=too-many-locals
-def signwriting_to_image(fsw: str, antialiasing=True, trust_box=True,
-                         line_color: tuple[int, int , int, int]=(0, 0, 0, 255), embedded_color=False) -> Image:
+# pylint: disable=too-many-locals, too-many-arguments
+def signwriting_to_image(fsw: str, antialiasing=True, trust_box=True, embedded_color=False,
+                         line_color: Tuple[int, int, int, int] = (0, 0, 0, 255),
```
Comment (AmitMY) [inline signwriting/visualizer/visualize.py]: perhaps it is worth, for readability:
```py
type RGBA = Tuple[int, int, int, int]
```

File: signwriting/visualizer/visualize.py [hunk 3]
```diff
@@ -53,7 +55,7 @@ def signwriting_to_image(fsw: str, antialiasing=True, trust_box=True,
         x, y = symbol["position"]
         x, y = x - min_x, y - min_y
         symbol_id = key2id(symbol["symbol"])
-        draw.text((x, y), symbol_fill(symbol_id), (255, 255, 255, 255), fill_font)
+        draw.text((x, y), symbol_fill(symbol_id), fill_color, fill_font, embedded_color=embedded_color)
```
Comment (AmitMY) [inline signwriting/visualizer/visualize.py]: can we name these variables? `fill=fill_color, font=fill_font`

File: signwriting/visualizer/test_visualize.py [hunk 4]
```diff
@@ -55,10 +54,9 @@ def test_image_small_box_is_corrected(self):
         assert_array_equal(np.array(image), np.array(reference_image))
 
     def test_image_with_line_and_embedded_color(self):
```
Comment (AmitMY) [inline signwriting/visualizer/test_visualize.py]: this test is now problematic as it tests too many things at once: line color, and transparent fill.
the name suggests it tests `embedded_color` but it doesn't

needs at least three tests:
1. embedded_color
2. line_color
3. fill_color
4. ideally, line and fill
5. ideally, line and embedded

**Review by AmitMY — CHANGES_REQUESTED** [review]
Interesting work! some changes requested

File: signwriting/visualizer/test_visualize.py (hunk 1 above)
Comment (bipinkrish) [reply signwriting/visualizer/test_visualize.py]: Stupid formatter

**Comment by bipinkrish** [comment]
Its done.

**Review by AmitMY — APPROVED** [review]

**Pull request merged**